File: benchmarks/runtime/closure_call.js
let n = 500000;
let i = 0;
let sum = 0;
let offset = 10;
const fn = (a) => {
  offset += a;
  return offset;
};
while (i < n) {
  sum += fn(i);
  i += 1;
}
if (sum <= 0) process.exit(1);
console.log(sum);

File: benchmarks/runtime/closure_call.lua
local n = 500000
local i = 0
local sum = 0
local offset = 10
local function fn(a)
  offset = offset + a
  return offset
end
while i < n do
  sum = sum + fn(i)
  i = i + 1
end
if sum <= 0 then os.exit(1) end
print(sum)
